examples/applications/slack/pipeline.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+topics:
+  - name: "input-topic"
+    creation-mode: create-if-not-exists
+pipeline:
+  - name: "convert-to-json"
+    type: "document-to-json"
+    input: "input-topic"
+    configuration:
+      text-field: "text"
+  - name: "submit to slack API"
```

Comment:
You can define the "url" as a secret 

```yaml
  - name: "submit to slack API"
    type: "http-request"
    configuration:
        url: "${secrets.slack.url}"
        body: '{"text":"hello from Langstream"}'
        output-field: "field"
        method: "POST"
```

Then in example  secrets.yaml you can add a section like:

```yaml
secrets:
    - id: slack
      data:
          - url:  "http://xxxxx"
```

Reply:
I think this inclkudes a few comments you made:
 url: "${globals.slack.url}${secrets.slack.token}"